ccd-definition/CaseField.json

```diff
@@ -179,6 +179,23 @@
     "FieldType": "Children",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "children1",
+    "Label": "Child",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "partyChildren",
```

Comment:
I think ChildrenParty would be a better name for this. Caps for the first letter of a defined type